README.md

```diff
@@ -137,9 +130,9 @@ env.close()
 
 Each `mani_skill2` environment supports different **observation modes** and **control modes**, which determine the **observation space** and **action space**. They can be specified by `gym.make(env_id, obs_mode=..., control_mode=...)`.
 
-The supported observation modes are `pointcloud`, `rgbd`, `state_dict` and `state`. Note that for the Maniskill 2022 Challenge, only `pointcloud` and `rgbd` are permitted.
+The bascially supported observation modes are `pointcloud`, `rgbd`, `state_dict` and `state`. Extra observation modes are `pointcloud_robot_seg` and `rgbd_robot_seg`, which include robot segmentation masks in addition. Note that for the Maniskill 2022 Challenge, only `pointcloud` and `rgbd` (and their `robot_seg` versions) are permitted.
```

Comment:
Typo also we should write

The basic observation modes supported are ... Additional observation modes which include robot segmentation masks are `pointcloud_robot_seg` and `rgbd_robot_seg`.

Also change Maniskill 2022 challenge to either 2023 or cal it Maniskill2 challenge ( I recommend we call it maniskill2 challenge and make it more of a long running thing)

Reply:
I will correct the typo. I will call it ManiSkill2 challenge.